### ISSUE TITLE
Val_accuracy is not changing at all. 

### ISSUE THREAD

**Issue by shreya-cmd** [xiaogangLi/tensorflow-VTN#2]

**Comment by xiaogangLi** [comment]
Please check if the validation clips are read correctly?

**Comment by poincarelee** [comment]
@xiaogangLi 
Hi,  I  have trained the hmdb51, and also found the val_accuracy is 0.0169787. The training loss keeps 3.93 and don't decrease.
![image](https://user-images.githubusercontent.com/38481402/183574216-f04937ba-518c-4569-9996-ae802d01623c.png)
Is anything wrong with the code?
